doc/src/manual/unicode-input.md

```diff
@@ -0,0 +1,66 @@
+# Unicode Input
+
+!!! warning
+
+    This table may appear to contain missing characters in the second column, or even
+    show characters that are inconsistent with the characters as they are rendered in
+    the Julia REPL. In these cases, users are strongly advised to check their choice
+    of fonts in their browser and REPL environment, as there are known issues with
+    glyphs in many fonts.
+
+```@eval
+#
+# Generate a table containing all LaTeX and Emoji tab completions available in the REPL.
+#
+
+function tab_completions(symbols...)
+    completions = Dict{String, Vector{String}}()
+    for each in symbols, (k, v) in each
+        completions[v] = push!(get!(completions, v, String[]), k)
+    end
+    return completions
+end
+
+function unicode_data()
+    file = normpath(JULIA_HOME, "..", "..", "doc", "UnicodeData.txt")
+    url = "http://www.unicode.org/Public/UCD/latest/ucd/UnicodeData.txt"
```

Comment:
That would certainly improve the reproducibility of builds. I guess we should also include that file in the tarballs? It would be weird not to be able to build Julia and its documentation offline from the full tarball with dependencies, just because of that missing file.